### PR TITLE
Fixes for mv performance tests

### PR DESCRIPTION
**Pull request by juliayakovlev**
Change aws subnet and security-group; 
New yaml for performance regression test with MV;
Add assertion message; 
tester.rows_to_list function as staticmethod

### CODE REVIEW

File: performance_regression_test.py [hunk 1]
```diff
@@ -165,6 +165,7 @@ def _get_total_ops(self):
         return self._stats['results']['stats_total']['op rate']
 
     def assert_mv_performance(self, ops_without_mv, ops_with_mv, failure_message):
+        self.log.debug('Assert performance results. Ops without MV: {0}; Ops with MV: {1}'.format(ops_without_mv, ops_with_mv))
```
[the remark this answers is not in the document]
Comment (juliayakovlev) [reply performance_regression_test.py]: I always count format strings in goal to be sure that the strings are inserted in the correct order

File: sdcm/tester.py [hunk 2]
```diff
@@ -969,6 +969,7 @@ def _check_build_started():
             if time.time() - start > seconds_to_wait:
                 raise Exception("View building didn't start in {} seconds".format(seconds_to_wait))
 
+    @staticmethod
     def rows_to_list(rows):
         new_list = [list(row) for row in rows]
         return new_list
```
[the remark this answers is not in the document]
Comment (juliayakovlev) [reply sdcm/tester.py]: local variable is removed

File: tests/perf-regression.100threads.30M-keys.yaml [hunk 3]
```diff
@@ -51,8 +51,8 @@ backends: !mux
         us_east_1:
```
Comment (roydahan) [inline tests/perf-regression.100threads.30M-keys.yaml]: Create a new YAML file and change the instance_type_db to i3.4xlarge.

File: tests/perf-regression.100threads.30M-keys.yaml [hunk 4]
```diff
@@ -10,9 +10,8 @@ user_prefix: 'perf-regression-VERSION'
 failure_post_behavior: destroy
 space_node_threshold: 644245094
 ami_id_db_scylla_desc: 'VERSION_DESC'
-ip_ssh_connections: 'private'
+#ip_ssh_connections: 'private'
```
Comment (amoskong) [inline tests/perf-regression.100threads.30M-keys.yaml]: Why do you comment this?  I saw you still use private ssh connection in the new file perf-regression-mv.100threads.30M-keys-i3.yaml

File: tests/perf-regression-mv.100threads.30M-keys-i3.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+n_db_nodes: 3
+n_loaders: 4
+n_monitor_nodes: 1
+user_prefix: 'perf-regression-VERSION'
+failure_post_behavior: destroy
+space_node_threshold: 644245094
+ami_id_db_scylla_desc: 'VERSION_DESC'
+ip_ssh_connections: 'private'
+instance_provision: 'spot_low_price'
+experimental: 'true'
```
Comment (amoskong) [inline tests/perf-regression-mv.100threads.30M-keys-i3.yaml]: do we need this line in performance testing?

Comment (juliayakovlev) [reply tests/perf-regression-mv.100threads.30M-keys-i3.yaml]: Yes. It must for materialized view test

File: tests/perf-regression.100threads.30M-keys.yaml (hunk 4 above)
Comment (juliayakovlev) [reply tests/perf-regression.100threads.30M-keys.yaml]: It's original version of this yaml file. I didn't change it

**Pull request merged**